docs/resources/glossary.md

```diff
@@ -64,7 +64,7 @@ description: >-
 | Symfony |  |
 | TCP | Transmission Control Protocol, a standard that defines how to establish and maintain a network conversation through which application programs can exchange data. |
 | TLS | Transport Layer Security |
-| Trivy |  |
+| Trivy | A simple and comprehensive vulnerability scanner for containers, suitable for CI (Continuous Integration) |
```

Comment:
Thanks for this, sometimes I toss a word into the glossary and forget to go grab the definition from my glossary (which is here, btw: https://docs.google.com/spreadsheets/d/1VX1URCt7sre1GsmYCtmLdy-IUcMZMfaL03zaRWgOpRY/edit#gid=0)